### PR TITLE
XMPP Newletter March 2023

### PR DESCRIPTION
**Pull request by Echolon**
# XMPP Newsletter

## This is the draft for the next release covering March 2023.

*Final release is planned for 5th of April 2023.*

Contributions are much appreciated. You don't need any technical knowledge to contribute. If you need help, please ask in the Communication-Team group chat: [commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org).

If you would like to add news, that's possible via:

- Using our [online Pad](https://pad.nixnet.services/oHnY_ZvLT8SoFyCqIC2ung)
- Adding your content to this pull request's thread (i.e. a comment below)
- Creating a pull request
- Posting something in our XMPP channel (xmpp:[commteam@muc.xmpp.org](mailto:commteam@muc.xmpp.org)?join)

It's really helpful if you try to give a short summary (<5 lines) of your contribution or of what the link you're posting is offering. You don't need to write in a formal tone - just be passionate about why this motivates you or about what you are working on!

XMPP Newsletter Timeline

Status:

* [x] Collection of news
* [ ] Review
* [ ] Blog post + Newsletter mail (use `tools/newsletter_email.py` after the post has been published)
* [ ] Social media
* [ ] Translations

### CODE REVIEW

File: content/blog/2023-04-05-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+## XSF Announcements
+
+- xmpp.org got a new software section! Looking for XMPP software, i.e. clients, servers, libraries, components, and tools? Check out [xmpp.org](https://xmpp.org/)'s new [software section](https://xmpp.org/software/), which lets you filter software by your own criteria. Looking for a client which works on Android and supports audio/video calls? Looking for a library that supports [XEP-0461: Message Replies](https://xmpp.org/extensions/xep-0461.html)? Just apply the filter and see what you get!
+
+{{< figure src="/images/newsletter/2023-03-05/xmpp-software-section.png" caption="xmpp.org's new software section" >}}
+
+- If you are interested to become an [XSF member](https://xmpp.org/community/membership/) you can [apply in this quarter until May 21th, 2023, 00:00 UTC](https://wiki.xmpp.org/web/Membership_Applications_Q2_2023).
```
Comment (cal0pteryx) [inline content/blog/2023-04-05-newsletter.md:18]:
```suggestion
- If you are interested to become an [XSF member](https://xmpp.org/community/membership/) you can [apply during this quarter until May 21th, 2023, 00:00 UTC](https://wiki.xmpp.org/web/Membership_Applications_Q2_2023).
```

File: content/blog/2023-04-05-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+- [XMPP Office Hours](https://wiki.xmpp.org/web/XMPP_Office_Hours): available on our [YouTube channel](https://www.youtube.com/channel/UCf3Kq2ElJDFQhYDdjn18RuA)
+- [Berlin XMPP Meetup (remote)](https://mov.im/?node/pubsub.movim.eu/berlin-xmpp-meetup): monthly meeting of XMPP enthusiasts in Berlin, every 2nd Wednesday of the month
+
+## Articles
+
+- Debian package maintainers give [a summary](https://xmpp-team.pages.debian.net/blog/2023/03/xmpp-whats-new-in-bookworm.html) of XMPP packages that will be available in the next Debian stable release (Debian 12 "Bookworm").
+- Cheogram Android [introduces Stickers](http://blog.jmp.chat/b/cheogram-android-stickers-2023).
+- IgniteRealtime: [Translations everywhere!](https://discourse.igniterealtime.org/t/translations-everywhere/92628)
+- [Here’s Why You Should Build Scalable and Concurrent Applications with Elixir](https://www.erlang-solutions.com/blog/heres-why-you-should-build-scalable-and-concurrent-applications-with-elixir/) by Cara May-Cole
```
Comment (cal0pteryx) [inline content/blog/2023-04-05-newsletter.md:43]: This has nothing to do with XMPP directly?

File: content/blog/2023-04-05-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+- Carl Schwan writes on the [Digital Market Act workshop in Brussels](https://carlschwan.eu/2023/03/02/digital-market-act-workshop-in-brussels/)
+{{< figure src="/images/newsletter/2023-04-05/dma_brussels.jpg" caption="Digital Market Act workshop in Brussels" >}}
+
+- The IETF has approved [publication of Messaging Layer Security (MLS)](https://www.ietf.org/blog/mls-secure-and-usable-end-to-end-encryption/), a new standard for end-to-end security that will make it easy for apps to provide the highest level of security to their users. End-to-end encryption is an increasingly important security feature in Internet applications. It keeps users’ information safe even if the cloud service they’re using has been breached.
+
+## Software news
+### Clients and applications
+
+- [Cheogram Android 2.12.1-4](https://git.singpolyma.net/cheogram-android/refs/2.12.1-4) released with support for sending message replies and reactions, swipe to reply, "note to self" contact, viewing vcard4 profiles, and occasional MUC ping.
```
Comment (cal0pteryx) [inline content/blog/2023-04-05-newsletter.md:52]:
```suggestion
- [Cheogram Android 2.12.1-4](https://git.singpolyma.net/cheogram-android/refs/2.12.1-4) has been released with support for sending message replies and reactions, swipe to reply, "note to self" contact, viewing vcard4 profiles, and occasional MUC pings.
```

File: content/blog/2023-04-05-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+### Servers
+
+- Metronome IM [4.0.2](https://github.com/maranda/metronome/releases/tag/v4.0.2) brings some bug fixes.
+
+### Libraries & Tools
+
+- [python-nbxmpp 4.2.1 and 4.2.2](https://dev.gajim.org/gajim/python-nbxmpp/-/blob/master/ChangeLog) have been released, which fix some bugs with python-nbxmpp's HTTP library.
+- [QXmpp 1.5.1](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.1) and [QXmpp 1.5.2](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.2) have been released in February and [QXmpp 1.5.3](https://github.com/qxmpp-project/qxmpp/releases/tag/v1.5.3) in March! They fix some important bugs.
+- [xmppbl.org](https://xmppbl.org) provides a Real-Time Block List (RTBL) service for XMPP.
+- [XmppDotNet](https://www.ag-software.net/2023/03/22/xmppdotnet-announcement/) is the new name and next generation of MatriX vNext XMPP library.
```
Comment (cal0pteryx) [inline content/blog/2023-04-05-newsletter.md:70]:
```suggestion
- [XmppDotNet](https://www.ag-software.net/2023/03/22/xmppdotnet-announcement/) is the new name and next generation of the MatriX vNext XMPP library.
```

**Review by cal0pteryx — APPROVED** [review]

**Comment by Echolon** [comment]
## Editorial closing of the XMPP Newsletter March '23 release

Many many thanks to everyone for their help in creation, review and translation!!!

This is a community effort, and we would like to thank translators for their contributions. Volunteers are welcome! Translations of the XMPP Newsletter will be released here (with some delay):

- English (original): [xmpp.org](https://xmpp.org/categories/newsletter/)
  - General contributors: **Adrien Bourmault (neox), Alexander "PapaTutuWawa", Arne, emus, Licaon_Kter, Ludovic Bocquet, melvo, MSavoritias (fae,ve), wurstsalat, XSF iTeam**
- French: [jabberfr.org](https://news.jabberfr.org/category/newsletter/) and [linuxfr.org](https://linuxfr.org/tags/xmpp/public)
  - Translators: **Adrien Bourmault (neox), alkino, anubis, Benoît Sibaud, bobble bubble, Nÿco, Pierre Jarillon, Ppjet6, Ysabeau**  
- German: [xmpp.org](https://xmpp.org/categories/newsletter/) and [anoxinon.de](https://anoxinon.de/blog/)
  - Translators: **Jeybe, wh0nix**
- Italian: [notes.nicfab.eu](https://notes.nicfab.eu)
  - Translators: **nicfab**
- Spanish: [xmpp.org](https://xmpp.org/categories/newsletter/)
  - Translators: **daimonduff, TheCoffeMaker**

- :heart: :fireworks:  Next release will be next year on **5th of May 2023**!

Have a good time until then and see you soon!

**Pull request merged**